cmd/logcli/main.go

```diff
@@ -165,49 +170,76 @@ func newLabelQuery(labelName string, quiet bool) *labelquery.LabelQuery {
 	}
 }
 
+func newSeriesQuery(cmd *kingpin.CmdClause) *seriesquery.SeriesQuery {
+	// calculate series range from cli params
+	var from, to string
+	var since time.Duration
+
+	q := &seriesquery.SeriesQuery{}
+
+	// executed after all command flags are parsed
+	cmd.Action(func(c *kingpin.ParseContext) error {
+
+		defaultEnd := time.Now()
+		defaultStart := defaultEnd.Add(-since)
+
+		q.Start = mustParse(from, defaultStart)
+		q.End = mustParse(to, defaultEnd)
+		q.Quiet = *quiet
+		return nil
+	})
+
+	cmd.Flag("since", "Lookback window.").Default("1h").DurationVar(&since)
+	cmd.Flag("from", "Start looking for logs at this absolute time (inclusive)").StringVar(&from)
+	cmd.Flag("to", "Stop looking for logs at this absolute time (exclusive)").StringVar(&to)
+	cmd.Flag("match", "eg '{foo=\"bar\",baz=~\".*blip\"}'").Required().StringsVar(&q.Matchers)
```

Comment:
Can you show me an example of how it work with multiple matches ? what's the cli look like for them ? is it space based or comma seperated ?

Reply:
I'm asking cause I see that `StringsVar` is an array.

Reply:
I'd also like to know that, but otherwise LGTM!

Nice work @yeya24

Reply:
![image](https://user-images.githubusercontent.com/25150124/78057522-d9fe7300-7354-11ea-9f17-603b9a5a4c63.png)
This works just the same like `promtool query series`

Reply:
Great. I won't hold up the PR if you don't want to add the docs, but I'd appreciate it :). Let me know.

Reply:
Will add it this afternoon. Will ping you when I finish

Reply:
Hey, I have updated the docs. PTAL @owen-d